meta/runtime.yml

```diff
@@ -1,44 +1,2 @@
 ---
 requires_ansible: '>=2.9.10'
-plugin_routing:
-  modules:
-    gcdns_record:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gcdns_zone:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gce:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gcp_backend_service:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gcp_forwarding_rule:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gcp_healthcheck:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gcp_target_proxy:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gcp_url_map:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
-    gcpubsub_facts:
```

Comment:
Ah, wait. This one needs to stay (until 2.0.0).

You could remove the module file though (it's a symlink anyway) and make it a deprecated redirect in here. That way no Ansible 2.9 user can use it (they shouldn't anyway), but it won't break for Ansible 2.11 users who still use it.

Reply:
@felixfontein done in https://github.com/ansible-collections/community.google/pull/6/commits/4ac6dca4534ef1ed950de1bb26b6c75d7ec0bf8f